config/capsule-answers.yaml

```diff
@@ -0,0 +1,5 @@
+---
+certs:
+  generate: false
+  deploy: true
+capsule: true
```

Comment:
Today, capsule-certs-generate indicates to include **--puppet "true"**, should we continue to include that or simply update the answers (or migration) to ensure the following, so that the user doesn't need to specify it:

```
capsule:
  puppet: true
```